test/behaviour/graql/GraqlSteps.java

```diff
@@ -565,9 +583,38 @@ public boolean check(Concept concept) {
         }
 
         public boolean check(Concept concept) {
-            return concept.isAttribute()
-                    && type.equals(concept.asAttribute().asRemote(tx()).getType().getLabel())
-                    && value.equals(concept.asAttribute().getValue().toString());
+            if (!concept.isAttribute()) {
```

Comment:
style note: when we can fit into the char limit on one line, we tend to do so and get rid of the {}. However, i there's any block in a series of `if... else if... else` that has to have a multiline {}, all of them use multiline {} to be consistent